File: cpp/02 Constructor , Copy Constructor and Case Study/CPP/CPP/CargoSystem.cpp
#include <iostream>
#include "Parcel.cpp" 
using namespace std;

void displayReceipt(Parcel p){
	cout<<"\n\nShipment Receipt"<<endl;
	cout<<"-------------------------"<<endl;
	cout<<"Receipt No. : "<<p.getId()<<endl;
	cout<<"Sender Name : "<<p.getSenderName()<<endl;
	cout<<"Sender Address : "<<p.getSenderAddress()<<endl;
	cout<<"Receiver Name : "<<p.getReceiverName()<<endl;
	cout<<"Receiver Address : "<<p.getReceiverAddress()<<endl;
	cout<<"Parcel Weight : "<<p.getWeight()<<" grams"<<endl;
	cout<<"Parcel Shiping Charges : "<<p.getFee()<<endl;
}

main() {
	int receiptnum ;
	int   weight, charges;
	string sname, saddress, rname, raddress; 
	
	cout<<"Enter receipt no.: ";
	cin>>receiptnum;
	cin.ignore();
	
	cout<<"Enter sender name: ";
	getline(cin, sname);
	
	cout<<"Enter sender address: ";
	getline(cin, saddress);
	
	cout<<"Enter receiver name: ";
	getline(cin, rname);
	
	cout<<"Enter receiver address: ";
	getline(cin, raddress);
	
	cout<<"Enter weight(in grams)) of parcel: ";
	cin>>weight;
	
	cout<<"Enter shipment charges: ";
	cin>>charges; 
	
	Parcel p;
	p.setId(receiptnum);
	p.setSenderName(sname);
	p.setSenderAddress(saddress);
	p.setReceiverName(rname);
	p.setReceiverAddress(raddress);
	p.setWeight(weight);
	p.setFee(charges);
	 
	displayReceipt(p);
	
	system("pause");
}

File: cpp/VIdeo 09-XX Interfaces With Java/Inheritance/CargoSystem.java
import java.util.Scanner;
public class CargoSystem{
 	public static void main(String args[]){
	 Scanner cIn = new Scanner(System.in);
	int receiptnum, weight, charges;
	String  sname, saddress, rname, raddress; 
	
	System.out.print("Enter receipt no.: ");
	receiptnum = Integer.parseInt(cIn.nextLine());
	 
	System.out.print("Enter sender name: ");
	sname = cIn.nextLine();
	
	System.out.print("Enter sender address: ");
	 
	saddress = cIn.nextLine();
	
	System.out.print("Enter receiver name: ");
	rname = cIn.nextLine();;
	
	System.out.print("Enter receiver address: ");
	raddress = cIn.nextLine();
	
	System.out.print("Enter weight(in grams)) of parcel: ");
	weight = Integer.parseInt(cIn.nextLine());
	
	System.out.print("Enter shipment charges: ");
	charges = Integer.parseInt(cIn.nextLine());
	Parcel p = new Parcel();
	p.setId(receiptnum);
	p.setSenderName(sname);
	p.setSenderAddress(saddress);
	p.setReceiverName(rname);
	p.setReceiverAddress(raddress);
	p.setWeight(weight);
	p.setFee(charges);
	 
	displayReceipt(p);
	
 }
 static void displayReceipt(Parcel p){
 	System.out.println("\n\nShipment Receipt");
	System.out.println("-------------------------");
	System.out.println("Receipt No. : "+p.getId());
	System.out.println("Sender Name : "+p.getSenderName());
	System.out.println("Sender Address : "+p.getSenderAddress());
	System.out.println("Receiver Name : "+p.getReceiverName());
	System.out.println("Receiver Address : "+p.getReceiverAddress());
	System.out.println("Parcel Weight : "+p.getWeight()+" grams");
	System.out.println("Parcel Shiping Charges : "+p.getFee());
 }
}